schemas/triniti/notify/mixin/browser-notification/latest.xml

```diff
@@ -20,6 +20,21 @@
         </description>
         <default>true</default>
       </field>
+      <field name="braze_campaign_id" type="string">
```

Comment:
@digitalartcrew are there any pattern constraints we can apply to these braze fields or is it not worth the hassle?

Reply:
One notes, it that some params such as `send_id` says optional string but when creating one it must abide by the following: 

```
API and API triggered campaigns that are sent as a broadcast will automatically generate a send identifier if a send identifier is not provided. If you want to specify your own send identifier, you will have to first create one through the [/sends/id/create endpoint](https://www.braze.com/docs/api/endpoints/messaging/send_messages/post_create_send_ids/). The identifier must be all ASCII characters and at most 64 characters long. You can reuse a send identifier across multiple sends of the same campaign if you want to group analytics of those sends together.
```

But no docs mentioning this for the others

Reply:
> @digitalartcrew are there any pattern constraints we can apply to these braze fields or is it not worth the hassle?

@gdbrown some sample values

variation id: `webPush-427`
campaign id: `6b497c89-ce4f-a6ef-d123-1ce8de15ad20`
segment id: `5b905829-ca4d-4cd4-9d27-9dae2b6552c7`

we probably could scope them after some investigation but i'm not sure it's worth it

Reply:
@efjacobson so no uuid on campaign/segment?  just stringy city

Reply:
@gdbrown yeah thats fine. for some reason i thought we'd have to pick a version but the generic 8-4-4-4-12 uuid pattern should be good